src/utils/date-range.js

```diff
@@ -1,26 +1,35 @@
-import fecha from 'fecha';
+const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```

Comment:
No need for this array. Could just use `new Intl.DateTimeFormat('en', {month: 'short'}).format(new Date())`